packages/fiori/bundle.esm.js

```diff
@@ -5,6 +5,7 @@ import * as defaultFioriTexts from "./dist/generated/i18n/i18n-defaults.js";
 import "./dist/Assets.js";
 
 import "./bundle.common.js";
+import "./dist/illustrations/AllIllustrations.js";
```

Comment:
`AllIllustrations` is already imported in `bundle.common.js`

Reply:
You're right, I fixed this, thanks